include/flecs.h

```diff
@@ -203,6 +203,9 @@ typedef struct ecs_ref_t ecs_ref_t;
 /** Type information */
 typedef struct ecs_type_info_t ecs_type_info_t;
 
+/** Table data */
+typedef struct ecs_data_t ecs_data_t;
```

Comment:
I think this type can remain private (see below). It is likely to change in future releases as there are a couple of improvements on the roadmap that may change how it relates to a table.